README.md

```diff
@@ -1,5 +1,5 @@
 [![Travis CI](https://travis-ci.org/Microsoft/sqltoolsservice.svg?branch=dev)](https://travis-ci.org/Microsoft/sqltoolsservice)
-[![AppVeyor](https://ci.appveyor.com/api/projects/status/github/{INSERTUSERHERE}/sqltoolsservice?svg=true&retina=true&branch=dev)](https://ci.appveyor.com/project/{INSERTUSERHERE}/sqltoolsservice)
+[![AppVeyor](https://ci.appveyor.com/api/projects/status/g48acrfwbx7pj452?svg=true)](https://ci.appveyor.com/project/kburtram/sqltoolsservice)
```

Comment:
This was failing for me with permissions denied issue. Any ideas why? Message:
`{"message":"Project not found or access denied."}`